deployment/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>io.quarkiverse.jooq</groupId>
         <artifactId>quarkus-jooq-parent</artifactId>
-        <version>999-SNAPSHOT</version>
+        <version>2.2.0-SNAPSHOT</version>
```

Comment:
This is actually something we do to make the release process easier.

Reply:
Hi Guillaume, ok if your tooling makes it easier with 999, I will change it. I just needed consistency to be able to build at all, see a comment below.

Reply:
Just a question: How does 999-SNAPSHOT make the release process easier?